Week3/homework/mongodb/index.js

```diff
@@ -25,27 +47,49 @@ async function findEpisodesExercises(client) {
    */
 
   // Find the title of episode 2 in season 2 [Should be: WINTER SUN]
+const findTitle = await client
+    .db("databaseWeek3")
+    .collection("bob_ross_episodes")
+    .findOne({ episode: "S02E02" });
 
   console.log(
-    `The title of episode 2 in season 2 is ${"TODO: fill in variable here"}`
+    `The title of episode 2 in season 2 is ${findTitle.title}`
   );
 
   // Find the season and episode number of the episode called "BLACK RIVER" [Should be: S02E06]
+const findEpisode = await client
+    .db("databaseWeek3")
+    .collection("bob_ross_episodes")
+    .findOne({ title: "BLACK RIVER" });
 
   console.log(
-    `The season and episode number of the "BLACK RIVER" episode is ${"TODO: fill in variable here"}`
+    `The season and episode number of the "BLACK RIVER" episode is ${findEpisode.episode}`
   );
-
+  
   // Find all of the episode titles where Bob Ross painted a CLIFF [Should be: NIGHT LIGHT, EVENING SEASCAPE, SURF'S UP, CLIFFSIDE, BY THE SEA, DEEP WILDERNESS HOME, CRIMSON TIDE, GRACEFUL WATERFALL]
-
+ const findTitles = await client
+    .db("databaseWeek3")
+    .collection("bob_ross_episodes")
+    .find({ elements: { $in: ["CLIFF"] } })
+    .toArray();
+  const findTitlesStringify = findTitles
+    .map((result) => result.title)
+    .join(", ");
   console.log(
-    `The episodes that Bob Ross painted a CLIFF are ${"TODO: fill in variable here"}`
+    `The episodes that Bob Ross painted a CLIFF are ${findTitlesStringify}`
   );
 
   // Find all of the episode titles where Bob Ross painted a CLIFF and a LIGHTHOUSE [Should be: NIGHT LIGHT]
-
+const findTitles2 = await client
+    .db("databaseWeek3")
+    .collection("bob_ross_episodes")
+    .find({ elements: { $in: ["CLIFF"], $in: ["LIGHTHOUSE"] } })
```

Comment:
```suggestion
    .find({ elements: { $in: ["CLIFF", "LIGHTHOUSE"] } })
```

I think you can combine the two `$in`s here